bitcoin-explained/breaking-down-taproot-activation-options.md

```diff
@@ -0,0 +1,1674 @@
+---
+title: "Breaking Down Taproot Activation Options"
+transcript_by: Sjors via rev.com
+revision_by: edilmedeiros
+media: https://youtu.be/mT0t8Jm0m5E?si=ITfvLBQkhA6KA7DG
+tags: ["bitcoin-core", "taproot", "soft-fork"]

```

Comment:
tags are important for content discovery. The tags that available in the dropdown in the review website are taken from https://btctranscripts.com/tags/. You can also find a read-friendly list [here](https://btctranscripts.com/status.json) - look for `["existing"]["tags"]`

`soft-fork` is not in our list of available tags, but [soft-fork-activation](https://btctranscripts.com/tags/soft-fork-activation/) is.

Reply:
Great to know, I'll ensure I check against this json.